src/webresource-handler/index.ts

```diff
@@ -36,15 +35,6 @@ export interface WebResourceHandler {
 	onPreRespond: (webResource: WebResource) => Promise<WebResource>;
 }
 
-export class WebResourceError extends TypedError {}
```

Comment:
This is gone because it doesn't really belong here. Each handler needs to be able to tell its own error typing independent really & pinejs should just forward them